.github/workflows/ci.yml

```diff
@@ -77,18 +77,19 @@ jobs:
       - name: Build
         run: |
           if [ "$RUNNER_OS" == "macOS" ]; then
-              export PATH="/usr/local/opt/llvm/bin:$PATH"
-              export LDFLAGS="-L/usr/local/opt/llvm/lib"
-              export CPPFLAGS="-I/usr/local/opt/llvm/include"
+              export PATH="/opt/homebrew/opt/llvm@17/bin:$PATH"
+              export LDFLAGS="-L/opt/homebrew/opt/llvm@17/lib"
+              export CPPFLAGS="-I/opt/homebrew/opt/llvm@17/include"
               llvm-config --version
           fi
           cabal build all
 
       - name: Run tests
         run: |
           if [ "$RUNNER_OS" == "macOS" ]; then
-              export PATH="/usr/local/opt/llvm/bin:$PATH"
-              export LDFLAGS="-L/usr/local/opt/llvm/lib"
-              export CPPFLAGS="-I/usr/local/opt/llvm/include"
+              export PATH="/opt/homebrew/opt/llvm@17/bin:$PATH"
+              export LDFLAGS="-L/opt/homebrew/opt/llvm@17/lib"
+              export CPPFLAGS="-I/opt/homebrew/opt/llvm@17/include"
+              llvm-config --version
```

Comment:
Remove this line, rest looks good!

Reply:
Ah. I just noticed the section above also has this line. Weird that it was missing here..